.github/workflows/lint_python.yml

```diff
@@ -0,0 +1,21 @@
+name: lint_python
+on: [pull_request, push]
+jobs:
+  lint_python:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - run: pip install bandit black codespell flake8 isort mypy pytest pyupgrade safety
+      - run: bandit --recursive --skip B101,B110 .
+      - run: black --check . || true
+      - run: codespell --ignore-words-list="ba,trough,ue" --skip="*demo.ipynb" || true
+      - run: flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+      - run: flake8 . --count --exit-zero --max-complexity=28 --max-line-length=204 --show-source --statistics
+      - run: isort --check-only --profile black . || true
```

Comment:
This check fails, primarily on tests.  However, the test modules do imports in a specific order so we can disable librosa's cache module at runtime.